Palm updtaed the Music Player to save the current location for long-form tracks like podcasts, audiobook etc.

The app will do this only for a select list genres. However, Palm left out the standard "Books" and "Books & Spoken" genres, which can also be used to tag audiobooks.

This patch adds support for the "Books" and "Books & Spoken" genres.
Index: /usr/palm/applications/com.palm.app.musicplayer/scripts/ReorderablePlaylist.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.musicplayer/scripts/ReorderablePlaylist.js
+++ /usr/palm/applications/com.palm.app.musicplayer/scripts/ReorderablePlaylist.js
@@ -389,6 +389,8 @@ var ReorderablePlaylist = Class.create({
 		var needsBookmark;
 		
 		switch (genre){
+			case "books":
+			case "books & spoken":
 			case "podcast":
 			case "speech":
 			case "spokenword":
